build.sbt

```diff
@@ -18,7 +18,7 @@ lazy val riddlIdeaPlugin: Project = Root(
     With.build_info,
     With.coverage(90),
     With.aliases,
-    With.riddl("0.54.1")
+    With.riddl("0.55.0-1-c185a0c5-20241121-1150")
```

Comment:
It's not a good idea to put that kind of version number in a build. At the very least, commit your changes and use the new commit number, like 0.55.0-2-ab83e423. The hash, ab83e423, for the commit is fictitious so use the one that you get when you commit your changes. 

The reason it is a bad idea is because it precludes team work. NOBODY but you on your machine could recreate the same version of RIDDL. but, if you commit your work, they can.

Reply:
The dynver.sbt plugin generates that version number. It does so by starting with a tag (0.55.0) appending how many commits past that you are at (1) and the commit hash of that commit (c184a0c5). If you’ve also made local changes, uncommitted, it appends the latest timestamp of the modified files (20241121-115, i.e. 1:15 am UTC last Thursday).
So, from that I surmised that you MUST have a local change. Had it just been 0.55.0-1-c185a0c5, I would not have made that comment because that’s the kind of version number generated when no changes have been made.